### PR TITLE
Multiple login

### PR DESCRIPTION
**Pull request by Xaviju**

### CODE REVIEW

**Comment by jespino** [comment]
Looks great, but I miss it in the partials/includes/modules/invitation-login-form.jade and in the partials/includes/modules/register-form.jade

File: app/partials/includes/modules/invitation-login-form.jade [hunk 1]
```diff
@@ -30,10 +30,17 @@ form.login-form
...
+        
+    .contrib-plugins-wrapper(ng-if="contribPlugins.length")
+        label(translate="LOGIN_COMMON.ALT_LOGIN")
+
+        fieldset(
+            ng-repeat="plugin in contribPlugins|filter:{type: 'auth'}"
+            ng-include="plugin.template"
+        )
 
     fieldset(ng-repeat="plugin in contribPlugins|filter:{type: 'auth'}", ng-include="plugin.template")
```
Comment (jespino) [inline app/partials/includes/modules/invitation-login-form.jade]: Looks like this line must be deleted, mustn't it?

**Comment by Xaviju** [comment]
Should be fixed now

**Comment by jespino** [comment]
I like it!

**Pull request merged**